PublicDateAtUSN: 2019-08-01 10:00:00 UTC
Candidate: CVE-2019-14235
CRD: 2019-08-01 10:00:00 UTC
PublicDate: 2019-08-02 15:15:00 UTC
References: 
 https://ubuntu.com/security/notices/USN-4084-1
 https://www.cve.org/CVERecord?id=CVE-2019-14235
Description:
 An issue was discovered in Django 1.11.x before 1.11.23, 2.1.x before
 2.1.11, and 2.2.x before 2.2.4. If passed certain inputs,
 django.utils.encoding.uri_to_iri could lead to significant memory usage due
 to a recursion when repercent-encoding invalid UTF-8 octet sequences.
Ubuntu-Description: 
Notes: 
Bugs: 
Priority: medium
Discovered-by:
Assigned-to:
CVSS:
 nvd: CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H [7.5 HIGH]


Patches_python-django:
upstream_python-django: needs-triage
precise/esm_python-django: DNE
trusty_python-django: ignored (end of standard support)
trusty/esm_python-django: ignored (end of ESM support, was needed)
esm-infra-legacy/trusty_python-django: needed
xenial_python-django: released (1.8.7-1ubuntu5.10)
esm-infra/xenial_python-django: released (1.8.7-1ubuntu5.10)
bionic_python-django: released (1:1.11.11-1ubuntu1.5)
esm-infra/bionic_python-django: released (1:1.11.11-1ubuntu1.5)
disco_python-django: released (1:1.11.20-1ubuntu0.2)
focal_python-django: released (1:1.11.22-1ubuntu1)
esm-infra/focal_python-django: released (1:1.11.22-1ubuntu1)
jammy_python-django: released (1:1.11.22-1ubuntu1)
kinetic_python-django: released (1:1.11.22-1ubuntu1)
lunar_python-django: released (1:1.11.22-1ubuntu1)
mantic_python-django: released (1:1.11.22-1ubuntu1)
noble_python-django: released (1:1.11.22-1ubuntu1)
oracular_python-django: released (1:1.11.22-1ubuntu1)
plucky_python-django: released (1:1.11.22-1ubuntu1)
questing_python-django: released (1:1.11.22-1ubuntu1)
devel_python-django: released (1:1.11.22-1ubuntu1)
